ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -34,16 +38,29 @@ def __post_init__(self):
             provider=self.implementation,
         )
 
-    def map_split(self, ds_split: datasets.Dataset, num_proc: int) -> datasets.Dataset:
+    def map_split(
+        self, ds_split: datasets.Dataset, num_proc: int, writer_batch_size: int
+    ) -> datasets.Dataset:
         print(f'TTS mapping "{self.column_name}" to "{self.audio_column_name}"...')
-        return ds_split.map(self._map_sample, num_proc=num_proc).cast_column(
+        return ds_split.map(
+            self._map_sample, num_proc=num_proc, writer_batch_size=writer_batch_size
+        ).cast_column(
             self.audio_column_name, datasets.Audio(sampling_rate=self.sample_rate)
         )
 
     def _map_sample(self, sample):
         # using a Jinja template for some added flexibility
         # The {{ var }} syntax is how Jinja denotes variables
-        text = jinja2.Template("{{" + self.column_name + "}}").render(**sample)
+        try:
+            text = jinja2.Template("{{" + self.column_name + "}}").render(**sample)
+        except TemplateError as e:
+            print(f"Error rendering template: {e}")
+            print(f"column_name: {self.column_name}")
+            print(f"sample keys: {list(sample.keys())}")
+            raise ValueError(
+                f"Template rendering failed. Make sure column_name exists in the sample."
+            ) from e
+
         text = text["text"] if isinstance(text, dict) else text
```

Comment:
nit and not related to this PR: it feels like this line is also not needed anymore since it can/should be folded into the `column_name` string (cc @juberti).

Reply:
agreed, we might even in the future allow for user-supplied templates which would remove the need for `column_name`